src/components/list/list.ts

```diff
@@ -56,6 +63,18 @@ export class MdListItem implements AfterContentInit {
 
   constructor(private _renderer: Renderer, private _element: ElementRef) {}
 
+  /** @internal */
+  handleFocus() {
+    console.log('focus');
```

Comment:
Whoops.  Will do.

On Mon, May 23, 2016 at 3:37 PM, Kara <notifications@github.com> wrote:

> In src/components/list/list.ts
> <https://github.com/angular/material2/pull/502#discussion_r64301388>:
>
> > @@ -56,6 +63,18 @@ export class MdListItem implements AfterContentInit {
> >
> >    constructor(private _renderer: Renderer, private _element: ElementRef) {}
> >
> > +  /** @internal */
> > +  handleFocus() {
> > +    console.log('focus');
>
> Remove the console.logs?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/angular/material2/pull/502/files/c4569f49304bd970403d10a1e86325d85ac641ce#r64301388>
>